source/reference/verification/hash.txt

```diff
@@ -0,0 +1,99 @@
+.. _c2c-verify-hash-comp:
+
+===========================
+Verify with Hash Comparison
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can verify sync by comparing MD5 hashes of collections synced from the
+source cluster to the destination cluster. While hash comparison ensures
+that the destination cluster has received all changes from the source, the
+:dbcommand:`dbHash` command locks the cluster, preventing additional writes
+until it completes.
+
+About This Task
+---------------
+
+Hash comparison is not possible with sharded clusters. It also does not
+work for standalone servers and replica sets that use MongoDB 4.4 or earlier
+releases, since the document field order can vary.
+
+If you need to sync a sharded cluster or an older release of MongoDB, or
+if the locks are unacceptable for your application workload, use a different
+:ref:`verification method <c2c-verify-method>`.
+
+Steps
+-----
+
+.. procedure::
+
+   .. step:: Retrieve the Source Hash
```

Comment:
Step 1 and 2 should be run concurrently to not waste time one commit is finished.